frontend/src/scenes/notebooks/Notebook/notebookLogic.ts

```diff
@@ -226,6 +230,12 @@ export const notebookLogic = kea<notebookLogicType>([
         saveNotebookSuccess: sharedListeners.onNotebookChange,
         loadNotebookSuccess: sharedListeners.onNotebookChange,
 
+        saveNotebookFailure: ({ errorObject }) => {
+            if (errorObject.code === 'conflict') {
+                actions.showConflictWarning()
+            }
+        },
```

Comment:
I _think_ we should instead do this in the `saveNotebook` call. Partly because there we can better control the error state, as well as reset the `notebook` to a null value or something which would stop it from flashing the old content. 

Locally this loads really fast but in prod it will definitely be slower so the old version will likely still be visible for a second or so which might be confusing...

Reply:
Ahh gotcha. Please ignore my comment above. This seems like a better approach